test-integration/operator-tests/src/test/java/org/infinispan/operator/AdvancedSetupBIT.java

```diff
@@ -75,11 +80,7 @@ static void deploy() throws Exception {
       log.info("Password: {}", pass);
 
       Https.doesUrlReturnOK("http://" + testServer.host() + "/ping").waitFor();
-
-      Service external = openShift.services().withName(appName + "-external").get();
-      hostName = external.getStatus().getLoadBalancer().getIngress().get(0).getHostname();
-
-      Https.doesUrlReturnCode("https://" + hostName + ":11222/console/welcome", 200).interval(TimeUnit.SECONDS, 10).waitFor();
+      Https.doesUrlReturnCode("https://" + hostName + "/console/welcome", 200).interval(TimeUnit.SECONDS, 10).waitFor();
```

Comment:
It would be nice to test via unsecured health status rather than console availability.
`GET /rest/v2/cache-managers/default/health/status`

Reply:
That's already done above by checking `WellFormed` condition which should be equivalent [1]. This is more of a safeguard. Routes and mainly LoadBalancers can take some time to start redirecting traffic.

[1] https://github.com/infinispan/infinispan-operator/blob/master/test-integration/operator-tests/src/test/java/org/infinispan/operator/AdvancedSetupBIT.java#L68.